src/logic/currentSession/store/reducer/currentSession.ts

```diff
@@ -17,7 +17,7 @@ export default handleActions(
     [UPDATE_VIEWED_SAFES]: (state, action) => {
       const safeAddress = action.payload
 
-      const newState = state.updateIn(['viewedSafes'], (prev) =>
+      const newState = state.updateIn(['viewedSafes'], (prev: string[] = []) =>
```

Comment:
Can't we just make sure that `viewedSafes` is set when we set an initial value?

Reply:
https://github.com/gnosis/safe-react/issues/1346